### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by skumarqa2010**
1. Code refactoring for event signup specs
2. Exclude one test from execution as there is a bug

### CODE REVIEW

File: cypress/integration/non-regression-tests-suite/event-signup-flow.spec.js [hunk 1]
```diff
@@ -565,63 +565,72 @@ describe("Verify page load " + new Date(), () => {
 			});
 	});
 	it('It navigates to selected "Online Events" page', function () {
+		let event_Name;
```
Comment (ethax-ross) [inline cypress/integration/non-regression-tests-suite/event-signup-flow.spec.js:568]: Can we keep these consistent with `camelCase`?

File: cypress/integration/non-regression-tests-suite/homepage.spec.js [hunk 2]
```diff
@@ -49,12 +49,12 @@ describe(`Home page tests : Tests execution date and time : ${new Date()}`, () =
 		cy.verifySocialMediaLink(4, Navlinks.youtube);
 	});
 
-	it.skip("Has no detectable a11y violations on load (filtering to only include critical impact violations)", () => {
+	/*it("Has no detectable a11y violations on load (filtering to only include critical impact violations)", () => {
```
Comment (ethax-ross) [inline cypress/integration/non-regression-tests-suite/homepage.spec.js:52]: Worth putting in a comment to say why this is disabled so we have it in the code?

Comment (skumarqa2010) [reply cypress/integration/non-regression-tests-suite/homepage.spec.js]: Issues have been fixed and test is not throwing accessibility violation error now. Therefore I've removed the skip tag from the tests.

File: cypress/integration/non-regression-tests-suite/event-signup-flow.spec.js (hunk 1 above)
Comment (skumarqa2010) [reply cypress/integration/non-regression-tests-suite/event-signup-flow.spec.js]: Resolved

**Review by ethax-ross — APPROVED** [review]

**Pull request merged**